### PR TITLE
[W6.4h][W11-B1]Alvin Yan

### PR DESCRIPTION
**Pull request by Aussiroth**

### CODE REVIEW

File: src/seedu/addressbook/data/person/Printable.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Printable.java:3]: This interface and its methods should have header comments. 
If not, developers who need to implement this in their class will not know what the expected behavior is.

File: src/seedu/addressbook/ui/Formatter.java [hunk 2]
```diff
@@ -34,6 +35,17 @@ public String format(String... messages) {
         return sb.toString();
     }
 
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+    public static String getPrintableString(Printable... printables){
```
Comment (okkhoy) [inline src/seedu/addressbook/ui/Formatter.java:41]: What would be a better place to have this method?

**Comment by okkhoy** [comment]
@Aussiroth 
Some comments added. Please ack & close the PR after reading comments.

**Comment by Aussiroth** [comment]
Noted, thanks.

**Pull request closed without merging**